server/resp/src/main/java/org/infinispan/server/resp/SubscriberHandler.java

```diff
@@ -72,18 +73,14 @@ public CompletionStage<Void> onEvent(CacheEntryEvent<Object, byte[]> entryEvent)
          byte[] key = KeyChannelUtils.channelToKey(unwrap(entryEvent.getKey()));
          byte[] value = entryEvent.getValue();
          if (key.length > 0 && value != null && value.length > 0) {
-            // *3 + \r\n + $7 + \r\n + message + \r\n + $ + keylength (log10 + 1) + \r\n + key + \r\n +
-            // $ + valuelength (log 10 + 1) + \r\n + value + \r\n
-            int byteSize = 2 + 2 + 2 + 2 + 7 + 2 + 1 + (int) Math.log10(key.length) + 1
-                  + 2 + key.length + 2 + 1 + (int) Math.log10(value.length) + 1 + 2 + value.length + 2;
+            // *3 + \r\n + $7 + \r\n + message + \r\n + $ + keylength + \r\n + key + \r\n +
+            // $ + valuelength + \r\n + value + \r\n
+            int byteSize = 2 + 2 + 2 + 2 + 7 + 2 + 1 + ByteBufferUtils.stringSize(key.length)
+                  + 2 + key.length + 2 + 1 + ByteBufferUtils.stringSize(value.length) + 2 + value.length + 2;
```

Comment:
Why did this need to be changed? log10 should work, is the new method supposedly faster?

Reply:
TBH it was just to have that in a single method. Haven't measured the difference between the two.